README.md

```diff
@@ -1,14 +1,29 @@
-NYPLBase
+#NYPLBase#
 ========
-This README is a work in progress...
+NYPLBase (or familiarly, "Base") provides a "base" for all NYPL web properties. Color palette, typeface, default styles, and a bit of structure is all built into this code package. (For those familiar with Twitter Bootstrap, it's like the Library's own Bootstrap but with fewer options.) 
 
+The idea is that anyone who's building a web project for the Library, whether one of our internal dev teams, or an external consultant group, can download and use this code as the style base, and not have to worry about getting our brand right--it's already in there. Even if you're not using the code, you can still look at the demo page as a style guide.
 
-NYPLBase (or familiarly, "Base") provides a "base" for all NYPL web properties. Color palette, typeface, default styles, and a bit of structure is all built into this code package. (For those familiar with Twitter Bootstrap, it's like the Library's own Bootstrap but with fewer options.) 
+At this moment, May 2018, only three applications are still using NYPLBase. They are Location-App (previous Location-prototype), Research Collections, and Staff Profiles. Along with some old drupal pages. The applications after these are using NYPL Design Toolkit.
```

Comment:
As of May 2018, only three applications are using NYPLBase: Locations App (previously Location-prototype), Research Collections, and Staff Profiles. Some of the older Drupal pages also use NYPLBase. Subsequent applications are using the NYPL Design Toolkit, with the idea that NYPL Design Toolkit will supersede NYPLBase as the default.